content/python/concepts/regex/terms/non-digit-class/non-digit-class.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: "\\D"
+Description: 'Matches any non-digit (0-9) in a given pattern.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Strings'
+  - 'Regular Expressions'
+  - 'Characters'
+CatalogContent:
+  - 'introduction-to-regular-expressions'
+  - 'paths/computer-science'
+---
+
+The `\D` character class matches any non-digit (0-9) in a given pattern. The inverse of this class is designated with a lowercase, `\d`, and will match any character that is a digit.
+
+## Syntax
+
+```pseudo
+a_given_time = r'\d:\d\d'
+```
+
+Used by itself, the `\d` class will match a single valid character inside the pattern.
```

Comment:
"single valid character" is somewhat vague and could apply to any character class.